Description/tr-TR.md

```diff
@@ -0,0 +1,89 @@
+# Türkçe
+
+Performans ve grafik optimizasyonlarına odaklanan Minecraft için Fabric tabanlı bir mod paketidir.
```

Comment:
"optimizasyonlarına" and rest of the sentence is totally okay, I'd like to suggest another word `iyileştirmelerine`

Reply:
Solved